tests/conftest.py

```diff
@@ -17,24 +18,167 @@
 import runhouse as rh
 from runhouse.globals import configs
 
-dotenv.load_dotenv()
+SSH_USER = "rh-docker-user"
+
+
+class TestLevels(str, enum.Enum):
+    UNIT = "unit"
+    LOCAL = "local"
+    MINIMAL = "minimal"
+    THOROUGH = "thorough"
```

Comment:
qq: what are you thinking for the difference of thorough vs maximal?

Reply:
Thorough means the complete set of fixtures we'd want to run to have thoroughly tested this code, e.g. what we'd run before release if we changed something significantly. Maximal is essentially every possible fixture that could be run, even superfluous ones (e.g. all the GPU ones). I don't foresee us running maximal practically ever, but it's there just in case. I'll put this into a testing README in another PR (with a few other testing features I'd like to add).